src/base/searchengine.h

```diff
@@ -132,6 +137,7 @@ private slots:
     bool m_searchStopped;
     QTimer *m_searchTimeout;
     QByteArray m_searchResultLineTruncated;
+    QList<QProcess*> m_downloaders;
```

Comment:
There may be a rare case of zombie creation if qBt exits without reading exit statuses from these processes. I think you must terminate them in the `SearchEngine::~SearchEngine()`. Moreover, probably these processes should be also killed by a timeout, which is likely a subject for a separate PR.